spec/mailers/users/reset_password_instructions_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe "User Reset Password Instructions", type: :mailer do
+  describe "#send_reset_password_instructions" do
+    context "user gets an email with instructions" do
+      let(:user) { create(:user) }
+
+      before(:each) do
+        @time = Time.now.in_time_zone("America/New_York")
+        user.send_reset_password_instructions
+      end
+
+      let(:mail) { ActionMailer::Base.deliveries.last }
+
+      it "sends an email with instructions" do
+        expiration_time = (@time + 6.hours).strftime("%I:%M %p on %m/%d/%Y %Z")
```

Comment:
Can you use `travel_to` to make the time explicit rather than relying on dynamic times? That's asking for trouble :smile: